### PR TITLE
Table: fix table style

### PR DESCRIPTION
**Pull request by isunkui**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/theme-default/src/table.css [hunk 1]
```diff
@@ -93,16 +93,15 @@
       padding-right: 18px;
       line-height: 40px;
       box-sizing: border-box;
+      overflow: hidden;
```
Comment (QingWei-Li) [inline packages/theme-default/src/table.css]: td 上不是有 text overflow 的样式了么

Comment (isunkui) [inline packages/theme-default/src/table.css]: @QingWei-Li th标题包含了子元素 div，th 的样式没有起到作用

Comment (QingWei-Li) [inline packages/theme-default/src/table.css]: 好我一会试试

Comment (QingWei-Li) [inline packages/theme-default/src/table.css]: 麻烦改成 inherit，下面也一样

Comment (isunkui) [inline packages/theme-default/src/table.css]: @QingWei-Li .caret-wrapper inherit 并不能解决问题如下图，div 继承th 的 overflow:hidden和直接写 overflow:hidden这个区别不大呀
qq20160923-0

Comment (QingWei-Li) [inline packages/theme-default/src/table.css]: 哦对

**Pull request merged**